src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -756,13 +756,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * GET /courses/:courseId/with-exercises-and-relevant-participations Get the "id" course, with text and modelling exercises and their participations It can be used only by
+     * GET courses/:courseId/with-exercises-and-relevant-participations Get the "id" course, with text and modelling exercises and their participations It can be used only by
```

Comment:
```suggestion
     * GET /courses/:courseId/with-exercises-and-relevant-participations Get the "id" course, with text and modelling exercises and their participations It can be used only by
```